examples/statistics/hist.py

```diff
@@ -46,7 +46,7 @@
 # edit the histogram to our liking. Let's change the color of each bar
 # based on its y value.
 
-fig, axs = plt.subplots(1, 2, figsize=(10, 5), tight_layout=True)
+fig, axs = plt.subplots(1, 2, tight_layout=True)
```

Comment:
I assumed it was somewhat intentional in this one (since the first figure wasn't square.)